readthedocs/api/utils.py

```diff
@@ -70,14 +71,15 @@ def _url_template(self, query, selected_facets):
         })
         return url_template + '?' + query_string
 
-    def _search(self, request, model, facets=None, page_size=20,
+    def _search(self, request, model, facets=None, page_size=20,  # pylint: disable=too-many-locals
```

Comment:
This could go on L75 or on a separate line after that I believe, to not break up args with a comment